examples/Extensions/AddPrices.php

```diff
@@ -0,0 +1,310 @@
+            'ad_schedules' => [
+                self::createAdScheduleInfo(
+                    DayOfWeek::SUNDAY,
+                    10,
+                    MinuteOfHour::ZERO,
+                    18,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::SATURDAY,
+                    10,
+                    MinuteOfHour::ZERO,
+                    22,
+                    MinuteOfHour::ZERO
+                )
+            ]
+        ]);
+
+        // Creates an extension feed item operation.
+        $extensionFeedItemOperation = new ExtensionFeedItemOperation();
+        $extensionFeedItemOperation->setCreate($extensionFeedItem);
+
+        // Issues a mutate request to add the extension feed item and print its information.
+        $extensionFeedItemServiceClient = $googleAdsClient->getExtensionFeedItemServiceClient();
+        $response = $extensionFeedItemServiceClient->mutateExtensionFeedItems(
+            $customerId,
+            [$extensionFeedItemOperation]
+        );
+        $extensionFeedItemResourceName = $response->getResults()[0]->getResourceName();
+        printf(
+            "Created extension feed item with resource name: '%s'.%s",
+            $extensionFeedItemResourceName,
+            PHP_EOL
+        );
+
+        return $extensionFeedItemResourceName;
+    }
+
+    /**
+     * Creates a new price offer with the specified parameters.
+     *
+     * @param string $header the header
+     * @param string $description the description
+     * @param int $priceInMicros the price in micros
+     * @param string $currencyCode the currency code
+     * @param int $unit the enum value of unit
+     * @param string $finalUrl the final URL
+     * @param null|string $finalMobileUrl the final mobile URL
+     * @return PriceOffer the created price offer
+     */
+    private static function createPriceOffer(
+        string $header,
+        string $description,
+        int $priceInMicros,
+        string $currencyCode,
+        int $unit,
+        string $finalUrl,
+        string $finalMobileUrl = null
+    ) {
+        $priceOffer = new PriceOffer([
+            'header' => new StringValue(['value' => $header]),
+            'description' => new StringValue(['value' => $description]),
+            'final_urls' => [new StringValue(['value' => $finalUrl])],
+            'price' => new Money([
+                'amount_micros' => new Int64Value(['value' => $priceInMicros]),
+                'currency_code' => new StringValue(['value' => $currencyCode])
+            ]),
+            'unit' => $unit
+        ]);
+
+        // Optional: Sets the final mobile URLs.
+        if (!is_null($finalMobileUrl)) {
+            $priceOffer->setFinalMobileUrls([new StringValue(['value' => $finalMobileUrl])]);
+        }
+        return $priceOffer;
+    }
+
+    /**
+     * Creates a new ad schedule info with the specified attributes.
```

Comment:
nit: "attribute" and "parameter" are used across this example in method comments, it would be better to keep it consistent